multimodal/src/autogluon/multimodal/optimization/utils.py

```diff
@@ -469,12 +473,61 @@ def apply_two_stages_lr(
     return optimizer_grouped_parameters
 
 
+def get_trainable_params_efficient_finetune(
+    norm_param_names: List[str],
+    efficient_finetune: Optional[str] = None,
+    trainable_param_names: Optional[List[str]] = None,
+):
+    """
+     Get the list of trainable parameters according to the provided efficient finetuning method.
+
+    Parameters
+    ----------
+    norm_param_names
+        The parameters associated with the normalization layers
+    efficient_finetune
+        Efficient finetuning strategy. Trainable parameters will be adjusted according to the method.
+    trainable_param_names
+        Initial specification of layers that should be trained.
+
+    Returns
+    -------
+    Get list of trainable parameter names according to the provided efficient finetuning method.
+    """
+    if not trainable_param_names:
+        trainable_param_names = []
+
+    if efficient_finetune == BIT_FIT:
+        trainable_param_names.append(".*bias*.")
+    elif efficient_finetune == NORM_FIT:
+        trainable_param_names.append(".*bias*.")
+        trainable_param_names += norm_param_names
+    elif efficient_finetune in [LORA, IA3]:
```

Comment:
Do we need to add new efficient finetune options in the tutorial here https://auto.gluon.ai/stable/tutorials/multimodal/customization.html?highlight=efficient_finetune?